flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -120,6 +120,9 @@
 	/** This determines if incremental checkpointing is enabled. */
 	private final TernaryBoolean enableIncrementalCheckpointing;
 
+	/** Thread number used to download from DFS when restore, default value: 1. */
+	private int numberOfRestoringThreads;
```

Comment:
i think this can also be final

Reply:
We introduced a setter for it: `setNumberOfRestoringThreads`, not to explode the number of constructor args like for `enableIncrementalCheckpointing`. At the moment `RocksDBStateBackend` basically serves as a configuration object and factory for the keyed and operator backends.